examples/server_overwrite/app.R

```diff
@@ -0,0 +1,66 @@
+library(shiny)
+library(shiny.router)
+
+# This generates menu in user interface with links.
+menu <- (
+  tags$ul(
+    tags$li(a(class = "item", href = route_link("/"), "Page")),
+    tags$li(a(class = "item", href = route_link("second"), "Second page")),
+    tags$li(a(class = "item", href = route_link("third"), "A third page")),
+    tags$li(a(class = "item", href = route_link("fourth"), "A fourth page"))
```

Comment:
What is the purpose of page no 4 in this example?

Reply:
This was case when there is no specified local server callback (but global does) and the page is not loaded on app start. Very similiar to the main page but worth to realize the behaviour.